content/11.interpretation.md

```diff
@@ -1,19 +1,17 @@
 ## Tip 9: Don't over-interpret predictions {#interpretation}
 
-Once we have trained an accurate deep learning model, we often want to use it to deduce scientific findings.
-In doing so, we need to take care to correctly interpret the model's predictions.
-Because deep models can be difficult to interpret intuitively, there is a temptation to anthropomorphize the models.
-We must resist this temptation.
-
-A common saying in statistics classes is "correlation doesn't imply causality".
-While we know that accurately predicting an outcome doesn't imply learning the causal mechanism, it can be easy to forget this lesson when the predictions are extremely accurate.
-A poignant example of this lesson is [@tag:predicting-pneumonia-mortality; @doi:10.1145/2783258.2788613].
-In this study, the authors evaluated the capacities of several models to predict the probability of death for patients admitted to an intensive care unit with pneumonia.
+Once we have trained an accurate deep learning model, we often want to use it to deduce relationships and inform scientific findings.
+However, in doing this, we need to be careful to correctly interpret the model's predictions.
+Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to anthropomorphize the models, and to overinterpret the predictions in indulgent and/or inaccurate ways.
```

Comment:
Note that this line the focus of #268, which takes offense at the use of "anthropomorphize" in this context. Maybe we should just remove the phrase and replace it with what you wrote:

```suggestion
Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to overinterpret the predictions in indulgent and/or inaccurate ways.
```